azure-pipelines/main.yml

```diff
@@ -0,0 +1,72 @@
+# Minimal Build pipeline
+# This allow us to generate wheels and upload that to a feed.
+
+variables:
+  artifactFeed:
+
+  WHEELS_DIR: /tmp/tensorflow_pkg
+  bazel.startup.opts: --batch_cpu_scheduling --io_nice_level 7
+  bazel.build.opts: --jobs 6 --local_ram_resources=HOST_RAM*0.8
+
+name: $(BuildDefinitionName)_$(SourceBranchName)_$(BuildID)
+stages:
+  - stage: Build
+    jobs:
+      - job: bazelBuild
+        timeoutInMinutes: 1440
+        cancelTimeoutInMinutes: 5
+        strategy:
+          matrix:
+            linux_python38:
+              imageName: 'ubuntu-20.04'
+              python.version: '3.8'
+              use.bazel.version: 3.1.0
+            macos1015_python38:
+              imageName: 'macOS-10.15'
+              python.version: '3.8'
+              use.bazel.version: 3.1.0
+        displayName: Bazel build
+        pool:
+          vmImage: $(imageName)
+        steps:
+          - template: templates/python-dev.yml
+          - template: templates/check-bazel.yml
+          - bash: |
+              pushd $BUILD_SOURCESDIRECTORY
+              yes "" | ./configure
+              popd
+            displayName: "Configure a default TF2 build"
+          - template : templates/wheel-version.yml
+          - bash: |
+              bazel $BAZEL_STARTUP_OPTS build $BAZEL_BUILD_OPTS //tensorflow/tools/pip_package:build_pip_package
```

Comment:
I assumed those two env vars are being set by ./configure? or who?
just commenting in case you forgot to set them here explicitly

Reply:
the best i can tell, nowhere in TF. people set them usually in user's .bazelrc. TF's .bazelrc does not define any policies w.r.t. parallelism and memory use, the best i can tell.

Reply:
However, the best i can tell, this is fairly pointless in case of Microsoft-hosted agents, as they are all nailed down to 2-cpu configurations....  So there's a very little chance it would have any material effect.